src/content/en/2024/third-parties.md

```diff
@@ -3,17 +3,205 @@
 title: Third Parties
 description: Third Parties chapter of the 2024 Web Almanac covering data of what third parties are used on the web and a deep dive into preventing them causing negative performance impacts.
 hero_alt: Hero image of Web Almanac characters plugging various things into a web page.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
```

Comment:
@turban1988 can you please provide a short bio about yourself?

Reply:
```suggestion
turban1988_bio: Prof. Dr.-Ing. Tobias Urban, Professor of Computer Science, specializing in Cybersecurity, at the [Westphalian University of Applied Sciences](https://www.en.w-hs.de/). His research focuses on IT security, data protection, and the impact of the GDPR on the Internet.
```